tasks/common_packages.yml

```diff
@@ -12,6 +13,8 @@
       - ctags
       - sudo
       - zsh
+      - aptitude
```

Comment:
Some debian based distributions only have apt-get, apt-cache.. installed and others only aptitude and it's nice to have the same set of command line tools installed on all system. At least until apt has a proper batch option(The apt(8) commandline is designed as an end-user tool and it may change behavior between versions. While it tries not to break backward compatibility this is not guaranteed either if a change seems beneficial for interactive use.)